hadoop-ozone/recon/src/main/java/org/apache/hadoop/ozone/recon/tasks/NSSummaryTaskDbEventHandler.java

```diff
@@ -40,6 +40,7 @@ public class NSSummaryTaskDbEventHandler {
 
   private static final Logger LOG =
       LoggerFactory.getLogger(NSSummaryTaskDbEventHandler.class);
+  public static final long NSSUMMARY_FLUSH_TO_DB_MAX_THRESHOLD = 150 * 1000L;
```

Comment:
check if this needs expose as configuration

Reply:
@sumitagrawl - This is exposed as configuration. Pls re-review.